### ISSUE TITLE
[KEYBOARD]: Direct Deposit - Trigger to enter information should be a button, must be keyboard focusable

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#1330]
## Description

The `<a>` tags to open the modal for entering direct deposit information are not keyboard focusable. They should be `<button>` tags and have some specific requirements for keyboard navigation. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As a keyboard user, I want the direct deposit calls to action to be keyboard focusable
* As this same user, I want the call to action I clicked on to receive keyboard focus again when I close the direct deposit modal

## WCAG or Vendor Guidance (optional)

* [Keyboard: Understanding SC 2.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html)

## Screenshots or Trace Logs

![Screen Shot 2019-08-27 at 3.50.21 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/b3b98926-ee1c-4dc2-9d0d-acc756128952)

**Comment by mojaray2k** [comment]
@1Copenut  Where is this page? I am looking at the screenshot but cannot find the page.

**Comment by Justin-Pickett** [comment]
@1Copenut , @mojaray2k  has corrected this issue. Can you validate this has been fixed? Thank you.

**Comment by 1Copenut** [comment]
@Justin-Pickett && @mojaray2k I re-tested this with Chrome on MacOS and verified the `<button>` elements were keyboard focusable, and responded to mouse, `ENTER`, and `SPACE`. Focus was restored on closure, so looks great. Moving this issue to closed.
